### PR TITLE
EAS configuration and AI Launchpad

### PR DESCRIPTION
**Pull request by vikas-thimmiaha**

### CODE REVIEW

File: src/api/.cdsrc.json [hunk 1]
```diff
@@ -1 +1,17 @@
...
+        "requires": {
+            "GENERATIVE_AI_HUB": {
+                "kind": "rest",
+                "credentials": {
+                    "destination": "SMART_CONVERTER_AI_CORE",
+                    "requestTimeout": "300000"
+                },
+                "RESOURCE_GROUP_ID": "default",
+                "DEPLOYMENTS": {
+                    "CHAT_COMPLETION_ID": "d6bb1fe78f97a16f"
```
Comment (kay-schmitteckert) [inline src/api/.cdsrc.json:12]: - please make this generic and put as "src/api/.cdsrc.sample.json"
- rename destination to GENERATIVE_AI_HUB"
- .cdsrc.json should be in gitignore

File: src/api/Makefile_20240222150714.mta [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Generated with Cloud MTA Build Tool version 1.2.19
```
Comment (kay-schmitteckert) [inline src/api/Makefile_20240222150714.mta:1]: - remove the Makefile

File: src/api/srv/converter.ts [hunk 3]
```diff
@@ -6,26 +6,26 @@ import { v4 as uuidv4 } from "uuid";
...
 const STOP_SEQUENCE: any = null;
 
 const GPT_PARAMS = {
-    engine: ENGINE,
     max_tokens: MAX_TOKENS,
     temperature: TEMPERATURE,
     frequency_penalty: FREQUENCY_PENALTY,
     presence_penalty: PRESENCE_PENALTY,
     stop: STOP_SEQUENCE
 };
```
Comment (kay-schmitteckert) [inline src/api/srv/converter.ts:21]: do we still need this?

File: src/mobile-app/package.json [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
-        "@expo/ngrok": "^4.1.0",
-        "@types/react": "~18.0.14",
-        "@types/react-native-get-random-values": "^1.8.0",
-        "@types/react-native-vector-icons": "^6.4.13",
-        "@types/uuid": "^9.0.1",
-        "prettier": "2.8.2",
-        "typescript": "^4.9.4"
-    },
-    "private": true
+  "name": "smart-converter",
```
Comment (kay-schmitteckert) [inline src/mobile-app/package.json:2]: - please change formatting to "4 spaces"

**Review by kay-schmitteckert — CHANGES_REQUESTED** [review]

File: src/api/srv/converter.ts (hunk 3 above)
Comment (vikas-thimmiaha) [reply src/api/srv/converter.ts:21]: Removed them. Tested locally without them and it worked.

**Review by kay-schmitteckert — APPROVED** [review]

**Pull request merged**